security.rst

```diff
@@ -1891,8 +1888,10 @@ as a routing resource:
     The :class:`Symfony\\Bundle\\SecurityBundle\\Routing\\LogoutRouteLoader` was
     introduced in Symfony 6.4.
 
-Another option is to create a route matching the name or URI you configured as
-``path``. You don't have to associate a controller because it wouldn't be called:
+Another option is to configure ``path`` as a route name, which can be useful if
+you want logout URIs to be translated according to the current locale e.g.
+In that case, you have to create this route yourself.
+Note that it doesn't need a controller, because it wouldn't be called anyways:
```

Comment:
This seems a true edge case, I would create a separate "Customizing the Logout Path" section after "Customizing Logout" with this content.